NorthstarDLL/plugins/plugin_abi.h

```diff
@@ -114,32 +110,7 @@ struct PluginEngineData
 	void* conVarRegister;
 	void* ConVar_Vtable;
 	void* IConVar_Vtable;
-};
-
-struct PluginGameStatePresence
-{
-	const char* id;
-	const char* name;
-	const char* description;
-	const char* password;
-
-	bool isServer;
-	bool isLocal;
-	GameState state;
-
-	const char* map;
-	const char* mapDisplayname;
-	const char* playlist;
-	const char* playlistDisplayname;
-
-	int currentPlayers;
-	int maxPlayers;
-
-	int ownScore;
-	int otherHighestScore; // NOTE: The highest score OR the second highest score if we have the highest
-	int maxScore;
-
-	int timestampEnd;
+	void* g_pCVar;
```

Comment:
Why expose this when you can run `CreateInterface` on `vstdlib` to get the same pointer

Reply:
it's just easier to expose it in `m_sEngineData` since it already has stuff, altought interfaces for plugins could be better in some ways.

Reply:
also I don't want to link rrplug to winapi yet

Reply:
"I don't want to link my personal project to winapi so i'm going to make the plugin api worse for everyone"

Reply:
how does it make it worse?

Reply:
Extra pointer in the struct for which there is an alternative to get it that's arguably better since it allows you to get any interface instead of hardcoding pointers to them.

Reply:
I would just keep it for this version since v2 already had hard coded stuff